wger/core/static/js/wger-core.js

```diff
@@ -697,4 +716,73 @@ $(document).ready(function () {
       '/' + token;
     window.location.href = targetUrl;
   });
+
```

Comment:
These are not your changes. When you git pull --rebase. Endeavour to squash and always have your commit at the top.